src/lib/compiled-aqua/installation-spell/cli.ts

```diff
@@ -5,15 +5,15 @@
  * This file is auto-generated. Do not edit manually: changes may be erased.
```

Comment:
I wonder if it is possible to avoid committing compiled aqua to the git. And instead compile Aqua on the fly.

wdyt?

Reply:
added a task for that https://linear.app/fluence/issue/DXJ-344/dont-store-compiled-aqua-under-version-control